joomla/joomla-cms
# Fahrenheit 451

#### Summary of Changes

If you activate advanced mode in the option for the com_redirect component you will see that you can set the Redirect Status Code to be used with each link

This PR adds the recently added 451 status code https://tools.ietf.org/html/rfc7725
#### Testing Instructions

Enable advanced mode
Create a redirect and you will see that you can now select 451
